state_transfer/transfer/blockrsync/blockrsync.go

```diff
@@ -0,0 +1,74 @@
+package blockrsync
+
+import (
+	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/konveyor/crane-lib/state_transfer/endpoint"
+	"github.com/konveyor/crane-lib/state_transfer/transfer"
+	"github.com/konveyor/crane-lib/state_transfer/transport"
+)
+
+const (
+	blockrsyncImage     = "quay.io/awels/blockrsync:latest"
```

Comment:
do you intend to keep the image in your account/

Reply:
No, I would love for this to be part of the konveyor organization. I just started the development in my local repo so I could make progress.